public/plugins/ckeditor/skins/bootstrapck/scss/browser-specific/ie8/editor_ie8.scss

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2003-2014, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+/*
+editor_ie8.css
+===============
+
+This file contains styles to used by Internet Explorer 8 only.
+*/
+
+/* Base it on editor_ie.css, overriding it with styles defined in this file. */
+@import "../../components/editor";
+
+.cke_toolbox_collapser .cke_arrow
```

Comment:
Selector `cke_toolbox_collapser` should be written in hyphenated BEM (Block Element Modifier) format<br>Selector `cke_arrow` should be written in hyphenated BEM (Block Element Modifier) format